seeder/seeder.go

```diff
@@ -101,20 +101,29 @@ func (s *StrategyRestSeeder) WithConfigManager(configManager CMRetrieve) *Strate
 	return s
 }
 
+func (s *StrategyRestSeeder) replaceAuthToken() error {
+	if s.configManager != nil && len(s.authInstructions) > 0 {
+		replacedAuthInstructions, err := configmanager.RetrieveMarshalledJson(&s.authInstructions, s.configManager, *s.configManagerOptions)
+		if err != nil {
+			return fmt.Errorf("Error while replacing secrets placeholders in authmap - %v", err)
+		}
+		s.rest.WithAuth(*replacedAuthInstructions)
+	}
+
+	return nil
+}
+
 // Execute the built actions list
 func (s *StrategyRestSeeder) Execute(ctx context.Context) error {
 	var errs []error
 	replacedActions := s.actions
 	// assign each action to method
 	s.log.Debugf("actions: %v", s.actions)
 	// configmanager the auth portion for any tokens
-	if s.configManager != nil && len(s.authInstructions) > 0 {
-		replacedAuthInstructions, err := configmanager.RetrieveMarshalledJson(&s.authInstructions, s.configManager, *s.configManagerOptions)
-		if err != nil {
-			return fmt.Errorf("Error while replacing secrets placeholders in authmap - %v", err)
-		}
-		s.rest.WithAuth(*replacedAuthInstructions)
+	if err := s.replaceAuthToken(); err != nil {
+		return err
 	}
+	
```

Comment:
we should do a similar thing for this bit [too](https://github.com/dnitsch/reststrategy/pull/25/files#diff-11dcf49fbe3e29c2b073b9ca5775012c49cb088ee6826cc73575a512ca14616dR130-R144)

Will need to set up a named loop for that though